tempesta_fw/http_sess.c

```diff
@@ -425,6 +433,13 @@ tfw_http_sticky_notfound(TfwHttpReq *req)
 	if (!tfw_cfg_sticky.enforce)
 		return TFW_HTTP_SESS_SUCCESS;
 
+	if (tfw_cfg_sticky.max_misses &&
+	    atomic_inc_return(&cli->miss_count) > tfw_cfg_sticky.max_misses)
```

Comment:
You can set ` tfw_cfg_sticky.max_misses` to `INT_MAX` for case when `max_misses` check is disabled. The approach is used for some other directives.

Reply:
Yes, but I choose Frang-like approach, since if we used `0` as working value of `tfw_cfg_sticky.max_misses`, then we would block the first redirected request (i.e. we would block all requests) which doesn't make sense, I think.